catalog/spatial/registry/registry-federation-admin-service-impl/src/main/java/org/codice/ddf/registry/federationadmin/service/impl/RefreshRegistryEntries.java

```diff
@@ -166,27 +223,44 @@ private void writeRemoteUpdates(List<Metacard> remoteMetacardsToUpdate)
                 return null;
             });
         } catch (PrivilegedActionException e) {
-            String message = "Error writing remote updates.";
-            LOGGER.debug("{} Metacard IDs: {}", message, remoteMetacardsToUpdate);
-            throw new FederationAdminException(message, e);
+            throw new FederationAdminException("Error writing remote updates", e);
```

Comment:
Any reason we got rid of the debug log? Wouldn't having the list of metacard IDs if this ever happens be useful? Assuming that this will be a very rare occurrence, shouldn't we even consider raising this to info or warn?